demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
+        value_type: 1 # STRING
+      - name: target.name
+        value_type: 1 # STRING
+      - name: api.name
```

Comment:
can we remove 'api' attributes until we get API management back? It's confusing since it's not clear whether this should be working or not.